BackEndApp/authv1/views.py

```diff
@@ -40,15 +59,19 @@ def login(request):
         {"message": message, "user": username, "token": token}, status=status
     )
 
-
 @api_view(["POST"])
 def register(request):
     try:
         username = request.data.get("username")
+        # username = "priyansh"
         email = request.data.get("email")
+        # email = "priyansh@gmail.com"
         password = request.data.get("password")
+        # password = "priyansh"
         first_name = request.data.get("first_name")
+        # first_name = "priyansh"
         last_name = request.data.get("last_name")
+        # last_name = "maheshwari"
```

Comment:
@priyanshm07 please remove the commented lines. There are few more files that have commented lines. Please remove all of them

Reply:
@RusherRG ok, am updating it.